src/sentry/static/sentry/app/views/projectDetail/projectDetail.tsx

```diff
@@ -168,7 +168,9 @@ const StyledPageContent = styled(PageContent)`
 `;
 
 const StyledSdkUpdatesAlert = styled(GlobalSdkUpdateAlert)`
-  margin-bottom: 0;
+  @media (min-width: ${p => p.theme.breakpoints[1]}) {
+    margin-bottom: 0;
+  }
 `;
```

Comment:
Damn, why do we lose the gap on smaller screens?

Reply:
Our `Layout.Body` component stops being a grid